rdmo/conditions/tests/test_viewset_condition.py

```diff
@@ -18,16 +18,19 @@
         'editor': 200, 'reviewer': 200, 'api': 200, 'user': 403, 'anonymous': 401
```

Comment:
These are the correct permissions, since detail views give 404 when the check fails.